android/KMEA/app/src/main/assets/keyboard.html

```diff
@@ -118,7 +118,7 @@
 
     function setKeymanLanguage(keyboardName, internalName, languageName, langId, kbdFile, font, oskFont, package) {
       var kmw=window['keyman'];
-      var kbdInterface=kmw['interface'];
+      var kbdInterface=window['KeymanWeb'];
```

Comment:
```suggestion
      var kbdInterface=window.KeymanWeb;
```

No need to use `window['..,']` if we aren't minifying.